src/backend/tests/models/charge_types/test_person_felony.py

```diff
@@ -0,0 +1,193 @@
+from expungeservice.models.expungement_result import EligibilityStatus
+from expungeservice.models.charge_types.person_felony import PersonFelonyClassB
+from expungeservice.models.charge_types.felony_class_c import FelonyClassC
+from tests.factories.charge_factory import ChargeFactory
+from tests.models.test_charge import Dispositions
+import pytest
+
+person_felony_statutes = [
```

Comment:
Similarly this list seems to be duplicated

Reply:
yep, see above